CrewChiefV4/CrewChief.cs

```diff
@@ -883,6 +886,7 @@ public Boolean Run(String filenameToRun, Boolean dumpToFile)
                             if (currentGameState.SessionData.IsNewSession)
                             {
                                 Console.WriteLine("New session");
+                                PlaybackModerator.clearVerbosityData();
```

Comment:
Jim, sorry for the nitpick but just the suggestion - we don't have common style in CC, and it is my fault as well, but at least I think we should try keeping one file consistent with its style.  So, we should name all PlaybackModerator methods so that their names start with either lower case or upper case.  Also, I like when I can see while reading code, if variable is member or local, so I always prefix with this. or type name.  It would be nice if all code within a file, and eventually in CC overall, uses same style.   Feel free to ignore this one, just sharing my perspective.

Reply:
good point